src/sass/_list-view.scss

```diff
@@ -0,0 +1,201 @@
+@use '_variables.scss' as *;
+
+.list-container {
+	display: grid;
+	place-items: center;
+	.first-line {
+		font-family: $header-font;
+		color: $light-green;
+		font-weight: 400;
+		font-size: 60px;
+		line-height: 72px;
+		text-align: center;
+	}
+	.second-line {
+		font-family: $body-font;
+		font-style: normal;
+		font-weight: 400px;
+		font-size: 40px;
+		line-height: 60px;
+		text-align: center;
+		color: #ff404a;
+	}
+
+	.add-button {
+		position: relative;
+		transition: 0.5s ease-out;
+		width: 153px;
+		height: 114px;
+		background: $yellow;
+		border-radius: 40px;
+		font-family: $header-font;
+		font-style: normal;
+		font-weight: 400;
+		font-size: 30px;
+		line-height: 36px;
+		text-align: center;
+		color: $brown;
+		&::before {
+			position: absolute;
+			bottom: -37px;
+			left: -36px;
+			content: url('/img/before.svg');
+		}
+		&::after {
+			position: absolute;
+			top: -9px;
+			right: -38px;
+			content: url('/img/after.svg');
+		}
+	}
+}
+
+//list search view (with token) list.js
+
+.item-list-container {
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+	justify-content: center;
+
+	form {
+		display: flex;
+		flex-direction: column;
+		align-items: center;
+
+		label {
+			font-family: $header-font;
+			font-style: normal;
+			font-weight: 400;
+			font-size: 30px;
+			line-height: 36px;
+			text-align: center;
+			margin-bottom: 1em;
+			color: $light-green;
+		}
+		input[type='text'] {
+			margin-bottom: 2em;
+			background: #fff;
+			padding: 1.5em;
+			width: 80%;
+			border-radius: 10em;
+			transition: box-shadow 300ms ease;
+			color: $brown;
+			font-family: $header-font;
+			font-size: 1.5em;
+		}
+	}
+	ul {
+		display: grid;
+		grid-template-columns: repeat(auto-fit, minmax(300px, 1fr));
+		grid-gap: 5em;
+		list-style: none;
+		padding: 0;
+		li {
+			text-decoration: underline;
+			color: #e5f4d3;
+			font-size: 3em;
+		}
+	}
+}
+
+.delete-button {
```

Comment:
I think that this could be a bit smaller. Can we also add a hover & focus state for #ally?